test/recipes/80-test_ssl_old.t

```diff
@@ -466,10 +468,15 @@ sub testssl {
 	plan tests => $protocolciphersuitcount + scalar(@protocols);
 
 	foreach my $protocol (@protocols) {
+	    my $protoarg = "";
+	    $protoarg = "-ssl3" if $protocol eq "SSLv3";
+	    $protoarg = "-dtls1" if $protocol eq "DTLSv1";
+	    $protoarg = "-dtls12" if $protocol eq "DTLSv1.2";
+
 	    note "Testing ciphersuites for $protocol";
 	    foreach my $cipher (@{$ciphersuites{$protocol}}) {
 		ok(run(test([@ssltest, @exkeys, "-cipher", $cipher,
-			     $protocol eq "SSLv3" ? ("-ssl3") : ()])),
+			     ($protoarg)])),
```

Comment:
I have no idea what you just said.  Are you assuming I know perl? :)

Will attempt this.